Commands/MessageCount/MessageCountSystem.go

```diff
@@ -5,13 +5,37 @@
 		-msgcount
 */}}
 
-{{$x:=dbIncr .User.ID "msgCount" 1}}{{$list:=""}}
-{{if reFind `\A-m(?:essages?|sg)?c(?:ount)?` .Message.Content}}
-	 You have typed {{$x}} messages.
-{{end}}
-{{if reFind `\A-t(?:op)?m(?:essages?|sg)?` .Message.Content}}
-	{{range dbTopEntries "msgCount" 10 0}}
-		{{- $list =print $list .User.Username " - " .Value "\n" -}}
-	{{end}}
-	{{sendMessage nil (cembed "title" "Message Count Leaderboard" "description" $list)}}
-{{end}}
+{{/* DONT TOUCH UNLESS YOU KNOW WHAT UR DOING c: */}}
+{{/* define variables */}}
+{{ $avatar := (joinStr "" "https://cdn.discordapp.com/avatars/" (toString .User.ID) "/" .User.Avatar ".png") }}
+{{ $x: = dbIncr .User.ID "msgCount" 1 }}{{ $list:="" }}
+{{ $embed := cembed "thumbnail" (sdict "url" $avatar) "title" "Message Counter" "description" (print .User.Mention " have sent " $x  " message(s) ") "color" 0xFF0000 }}
+{{ if reFind `\A\-m(?:essages?|sg)?c(?:ount)?` .Message.Content }} {{/* msgcount command */}}
+	{{ sendMessage nil $embed }}
+{{ end }}	
+{{ if reFind `\A\-t(?:op)?m(?:essages?|sg)?` .Message.Content }} {{/* topmessages command */}}
```

Comment:
yummy internal comments, ty
regex again with just (?i) at the start or crazy mode with (?i)\A(?:\-|<@!?204255221017214977>)\s*t(?:op)-?m(?:sgs?|essages?)?(?:\s|\z)

btw, we use \- instead of just - cus people tend to change their prefixes and, for example in case of ?, the regex would break so it's a bit of a safety precaution to save yaggy vs some work on  relatively simple fixes